README-new.md

```diff
@@ -0,0 +1,8 @@
+wp-cli/wp-cli-bundle
+=================
+
+Combines the most common commands into the standard, installable version of WP-CLI.
+
+Generally, bundled commands either relate directly to a WordPress API or offer some common developer convenience. New commands are included in the WP-CLI bundle when the [project governance](https://make.wordpress.org/cli/handbook/contributions/governance/) decides they should be. There isn't much of a formal process to it, so feel free to ask if you ever have a question.
+
+The handbook documents the [various ways you can install the bundle](https://make.wordpress.org/cli/handbook/guides/installing/). The Phar is [built on every merge](https://github.com/wp-cli/wp-cli-bundle/blob/main/.github/workflows/deployment.yml) and pushed to [wp-cli/builds](https://github.com/wp-cli/builds) repository. A stable version is [tagged a few times each year](https://make.wordpress.org/cli/handbook/contributions/release-checklist/).
```

Comment:
@schlessera In the original issue, you mentioned:

> What is the special versioning behavior of the bundle (mirroring `wp-cli/wp-cli` package)?

Can you clarify what you were referring to in this regard (and potentially add a suggestion covering what you want)?

Reply:
This was specifically referring to the fact that we do not have intermediate tagged releases for the `wp-cli/wp-cli` & `wp-cli/wp-cli-bundle` packages like with other commands, but rather that we have a stable release milestone that is then tagged once everything was successfully assembled. In the past, I tried to keep `wp-cli/wp-cli` & `wp-cli/wp-cli-bundle` in sync (this is the mirroring I mentioned), because one used the `VERSION` information from the other. This is a bit of a bad coupling between these two, but it also stems from the fact that `wp-cli/wp-cli` has a second responsibility in that it keeps backwards compatibility going for commands that depend on the framework.

How about adding something like this:
```suggestion
The handbook documents the [various ways you can install the bundle](https://make.wordpress.org/cli/handbook/guides/installing/). The Phar is [built on every merge](https://github.com/wp-cli/wp-cli-bundle/blob/main/.github/workflows/deployment.yml) and pushed to [wp-cli/builds](https://github.com/wp-cli/builds) repository. A stable version is [tagged a few times each year](https://make.wordpress.org/cli/handbook/contributions/release-checklist/).
Both `wp-cli/wp-cli` and `wp-cli/wp-cli-bundle` are milestoned and tagged for the next stable release. For `wp-cli/wp-cli` this represents the version of the WP-CLI framework, whereas it represents the WP-CLI Phar version for `wp-cli/wp-cli/bundle`. For backwards compatible reasons and to avoid causing confusion with third-party commands, we keep both of these versions in sync - whereas all the command repositories that make up the bundle are versioned independently according to semantic versioning principles as needed.
```
Hmm, I'm pretty sure this can be phrased in a better way. But does this communicate what I mean by "mirroring"?

Reply:
> But does this communicate what I mean by "mirroring"?

@schlessera Yep! I rephrased a bit and committed your suggestion.